qemu/tests/x86_cpu_flags.py

```diff
@@ -36,7 +36,14 @@ def run(test, params, env):
     error_context.context("Try to log into guest", logging.info)
     session = vm.wait_for_login()
     if params["os_type"] == "linux":
+        check_guest_cmd = params.get("check_guest_cmd")
```

Comment:
I will prefer `check_mds_cmd`, to be more accurate.

Reply:
> I will prefer `check_mds_cmd`, to be more accurate.

There is another case also will use this step and it will  be better using one common parameter. thanks.

Reply:
@nanliu-r Okay make sense.